packages/migration/migrations/hearth/20230612060923-update-patient-identifier-with-codeableconcept.js

```diff
@@ -66,7 +75,12 @@ export const up = async (db, client) => {
               }
             ]
             await db.collection('Patient').updateOne(
-              { _id: patient._id },
+              {
+                _id: patient._id,
+                'identifier.type': {
+                  $in: EVENT_IDENTIFIER
+                }
+              },
               {
                 $set: {
                   'identifier.$[].type': {
```

Comment:
@kayumuzzaman I'm still skeptical about this part. Probably the better way to do this would be to do it like this:
```ts
const updateIdentifiers = patient.identifier.map((identifier) => // return the updated identifier)
await db.collection('Patient').updateOne({_id: patient._id }, { $set { identifier: updatedIdentifiers }) 
```